setup.cfg

```diff
@@ -1,4 +1,12 @@
 [flake8]
-; E,W - disable pycodestyle checks as they may conflict with black
-ignore = E,W
+; E,W - disable flake8 checks that may conflict with black
+; https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8
+extend-ignore = E203, E501
+max-doc-length = 79
 
+[mypy]
```

Comment:
I believe `mypy` has support for `pyproject.toml`. Why not move it there?

Reply:
Sure, good point!